src/components/Contentful/StoryCard.vue

```diff
@@ -1,23 +1,58 @@
 <template>
-	<div class="tw-col-span-12">
-		<h4>{{ kickerHeadline }}</h4>
-		<dynamic-rich-text class="tw-text-center" :html="cardContent" />
+	<section
+		class="tw-h-full"
+		:class="themeClass"
+		v-if="hasBackgroundImage"
+	>
+		<kv-contentful-img
+			class="tw-h-full tw-object-cover"
+			:width="520"
+			fallback-format="jpg"
+			:contentful-src="backgroundImage.url"
+			:alt="backgroundImage.description"
+		/>
+		<p class="story-card__imageCard-title tw-text-h4">
+			{{ backgroundImage.title }}
+		</p>
+	</section>
+
+	<section
+		class="
+			story-card
+			tw-h-full
+			tw-bg-brand-200
+			tw-text-primary
+			tw-flex
+			tw-flex-col
+			tw-justify-between
+			tw-items-center
+			tw-p-6
+		"
+		:class="themeClass"
+		v-else
+	>
+		<p class="tw-text-center tw-text-h4">
+			{{ kickerHeadline }}
+		</p>
+		<dynamic-rich-text
+			class="story-card__content tw-text-center tw-h-full"
+			:html="cardContent"
+		/>
 		<dynamic-rich-text class="tw-text-center" :html="footer" />
-		{{ theme }}
-	</div>
+	</section>
 </template>
 
 <script>
 import { richTextRenderer } from '@/util/contentful/richTextRenderer';
 import DynamicRichText from '@/components/Contentful/DynamicRichText';
-
+import KvContentfulImg from '@/components/Kv/KvContentfulImg';
```

Comment:
For the followup to this, please refactor this to use the KvContentfulImg component from:
`import KvContentfulImg from '~/@kiva/kv-components/vue/KvContentfulImg';`